### DISCUSSION TITLE
出错了您稍后再试

### DISCUSSION THREAD

**Discussion by 420xincheng** [ChatGPTNextWeb/NextChat#332] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/74346123/229301498-6ab8ade4-cd1d-4ba8-8a05-4d74fb3e2460.png)
可能原因是啥？

**Comment by Bridge619** [comment, marked as answer]
免费的API没了？

**Comment by zhonghanwen** [comment]
遇到一样的问题
